galpy/potential/plotEscapecurve.py

```diff
@@ -145,14 +148,25 @@ def calcEscapecurve(Pot,Rs):
     except TypeError:
         grid=1
         Rs= nu.array([Rs])
+    try:
```

Comment:
I don't think it makes sense to allow an array for ``t`` here, because the point of the function is to compute the escape curve at a given time. So just change this to ``t`` being a number.